<launch>
    <node name="orb_slam2_gen_cfg" pkg="orb_slam2_ros" type="orb_slam2_gen_cfg"
          output="screen" cwd="node">
        <!--camera configuration file to be generated-->
        <param name="filename" value="$(find orb_slam2_ros)/cfg/default.yaml"/>
        <!--camera info topic name-->
        <param name="camInfoTopic" value="/camera/color/camera_info"/>
        <!--baseline is 55mm for D415 and 50mm for D435-->
        <param name="camBaseline" value="50"/>
        <!--Color order of the images (RGB or BGR)-->
        <param name="bRGB" value="true"/>
    </node>
</launch>